README.md

```diff
@@ -47,17 +43,22 @@ SWRS_ADDR_PBATTLEMGR = 0x0047579c
 ```
 
 ## Building from source
+The mod can be buit with `cargo` using the commands below.
+```bash
+rustup default nightly-2024-06-18-x86_64-pc-windows-msvc
+rustup component add rust-src --toolchain nightly-2024-06-18-x86_64-pc-windows-msvc 
```

Comment:
The platform of the toolchain needs not to be same with the target one. Simply `nightly-2024-06-18` is enough and also suitable for crosscompile.

Reply:
👍